### ISSUE TITLE
AttributeError: 'Dot' object has no attribute '_repr_svg_'

### ISSUE THREAD

**Issue by lijipeng787** [microsoft/tensorwatch#52]
my usage like this:
model = Model()
tw.draw_model(model, [1, 2, 256, 256])

and get error message:
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~/miniconda2/envs/pytorch_env/lib/python3.6/site-packages/IPython/core/formatters.py in __call__(self, obj)
    343             method = get_real_method(obj, self.print_method)
    344             if method is not None:
--> 345                 return method()
    346             return None
    347         else:

~/miniconda2/envs/pytorch_env/lib/python3.6/site-packages/tensorwatch/model_graph/hiddenlayer/pytorch_draw_model.py in _repr_svg_(self)
     11     def _repr_svg_(self):
     12         """Allows Jupyter notebook to render the graph automatically."""
---> 13         return self.dot._repr_svg_()
     14     def save(self, filename, format="png"):
     15         # self.dot.format = format

AttributeError: 'Dot' object has no attribute '_repr_svg_'

**Comment by BugKid** [comment]
I also encountered this problem, have you solved it?

**Comment by cloudpanl** [comment]
This issue has been fixed, modifying pytorch_draw_model.py like this can be solved. [#51](https://github.com/microsoft/tensorwatch/pull/51/files)

**Comment by pranavsekhar** [comment]
Hm, the fix in #51 isn't working for me. Did it work for either of you?

**Comment by BugKid** [comment]
> Hm, the fix in #51 isn't working for me. Did it work for either of you?

Yes, it works for me. Maybe you should try to restart your jupyter notebook or just reimport tensorwatch after modifying pytorch_draw_model.py

**Comment by sytelus** [comment]
I debugged this issue and it appears that pydot needs to be updated to resolve this error (see [this comment](https://github.com/pydot/pydot/issues/203#issuecomment-582766169)). Unfortunately, pydot seems like an unmaintained project so I forked pydot and applied open PRs against it which resolves this issue. If you are facing this issue, I would suggest to reinstall pydot from my fork: https://github.com/sytelus/pydot.

TensorWatch is updated to use pydot from above fork however it will only affect new installations because pip doesn't detect vertion changes from package installed from git URLs.

So in nutshell, do this:

```
pip uninstall pydot
pip install git+https://github.com/sytelus/pydot@v1.5.0#egg=pydot
```
